sys/net/gnrc/link_layer/gnrc_mac/internal.c

```diff
@@ -143,7 +143,7 @@ bool gnrc_mac_queue_tx_packet(gnrc_mac_tx_t* tx, uint32_t priority, gnrc_pktsnip
     int neighbor_id;
 
     /* Check whether the packet it for broadcast */
-    if (gnrc_netif_hdr_get_flag(pkt)&GNRC_NETIF_HDR_FLAGS_BROADCAST) {
+    if (gnrc_netif_hdr_get_flag(pkt) & (GNRC_NETIF_HDR_FLAGS_MULTICAST | GNRC_NETIF_HDR_FLAGS_BROADCAST)) {
```

Comment:
Yes, this has been tackled by [PR6627](https://github.com/RIOT-OS/RIOT/pull/6627), and should not be here after rebased.